README.md

```diff
@@ -78,6 +78,14 @@ There are many more parameters you can use. Display the usage doc with the follo
 
     GitLink.exe -h
 
+### Native PDBs
+
+Native PDBs (from C++ projects) are supported by using -a option:
+
+    GitLink.exe <nativePdbfile> -a
+
+All source code files from your git depot will be indexed in the PDB.
```

Comment:
@GeertvanHorrik in the meantime I'm a bit troubled by this `-a` switch since if you ship it, you'll be under pressure to maintain it, even though it's a scaling bug by design and eventually shouldn't be necessary to actually support `C++` projects.